oss-internship-2020/libarchive/ld_preload_example/minitar_main.cc

```diff
@@ -0,0 +1,195 @@
+/*-
```

Comment:
If this file is just the plain `minitar.c`, we should not duplicate it and instead reuse the one that ships with libarchive.